### PR TITLE
new contributor

### PR DESCRIPTION
**Pull request by marqh**
CLA recieved and processed

### CODE REVIEW

File: contributors.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
 { "profile_name":"jbednar", "organisation":"Continuum Analytics"},
 { "profile_name":"rrenfrow86", "organisation":"Continuum Analytics"},
 { "profile_name":"mir06", "organisation":"Wegener Center for Climate and Global Change"},
-{ "profile_name":"timmie", "organisation":""}
+{ "profile_name":"timmie", "organisation":""},
+{ "profile_name":"ltiao", "organisation":""},
```
Comment (pelson) [inline contributors.json]: Not valid JSON again...

Comment (pelson) [inline contributors.json]: We should add some CI on this repo to catch these cases.

**Comment by QuLogic** [comment]
Ping?

**Comment by bjlittle** [comment]
@marqh Ping ...

**Comment by marqh** [comment]
> Ping?
> @marqh Ping ...

sorry :$

updated

**Pull request merged**